app/views/posts/show.html.erb

```diff
@@ -1 +1,8 @@
-<h1>Show one specified post</h1>
\ No newline at end of file
+<section class="section">
+    <%= render 'post', post: @post %>
+    <% if @post.comments_counter > 0 %>
+          <% @post.comments.each do |comment| %>
+            <%= render 'comments', comment: comment, user: @user %>
+            <% end %>
+          <% end %>
+</section>
```

Comment:
- [ OPTIONAL ] Comments should have its own border as on the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png)

Yours:
![image](https://user-images.githubusercontent.com/62705947/236012485-dc4677fa-04cb-481e-950d-3671f1d85ee8.png)


Wireframe:
![image](https://user-images.githubusercontent.com/62705947/236012526-0f5d59a5-6654-4388-91df-854d90529005.png)